### PR TITLE
Pull request for #28: Command Line argument example

### PR DESCRIPTION
**Pull request by lboxell**

### CODE REVIEW

File: source/analysis/SConscript [hunk 1]
```diff
@@ -3,4 +3,4 @@ Import('env')
 env.BuildStata(target = ['#build/analysis/plot.eps',
                          '#build/analysis/table.txt'],
                source = ['#source/analysis/analysis.do',
-                         '#build/data/data.txt'])
\ No newline at end of file
+                         '#build/data/data_1.txt'])
```
Comment (arosenbe) [inline source/analysis/SConscript]: Can you add a carriage return?

**Comment by lboxell** [comment]
@arosenbe,

I implemented your comments.  Let me know if there is anything else. I'll wait to merge until #29 finishes because this branch requires https://github.com/gslab-econ/gslab_python/pull/50 to run properly.

**Comment by arosenbe** [comment]
@lboxell the example change looks good. It does seem like you might be using tabs in spaces where you meant spaces (see build.R but maybe other places too). Can you please check?

**Comment by lboxell** [comment]
@arosenbe Done.

**Pull request merged**